index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nelson Araujo's Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <section class="mobile-nav">
+            <div class="landing-header">
+                <div class="header-image"></div>
+                <button id='open-menu-btn' class="menu-btn">
+                    <img src="./assets/ic_menu.svg">
+                </button>
```

Comment:
[OPTIONAL] In line with the need for quality UX, I also suggest you add an aria-control attribute to the wrapper to make it clear to screen readers that contents are hidden. see [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls#:~:text=The%20aria%2Dcontrols%20attribute%20identifies%20the%20element%20(or%20elements)%20whose%20contents%20or%20presence%20are%20controlled%20by%20the%20element%20on%20which%20the%20attribute%20is%20set%2C%20regardless%20of%20what%20type%20of%20interaction%20initiates%20the%20impacted%20behavior.)